### ISSUE TITLE
Would it be interesting to implement BSP?

### ISSUE THREAD

**Issue by joostheijkoop** [mvndaemon/mvnd#485]
Dear authors,

Your project looks very promising, I have always been lacking such an optimization while working on Java projects.
It could be very nice if Intellij would be able to interface with the daemon, because I suspect, this could greatly improve the developer experience.
The Scala build tools have come to converge on BSP (https://build-server-protocol.github.io/) to allow for easier integration with Intellij and other editors/IDEs and seriously speed up the editors reaction time (multiple orders of magnitude for some cases)

I understand this is no easy feat and not really the current use case, but you might be interested in knowing BSP exists.

Cheers,

Joost

**Comment by gnodet** [comment]
Yes, that definitely looks interesting.  Any idea where the default maven provider is located ?
